test/babel-config.js

```diff
@@ -50,7 +50,7 @@ test('uses babelConfig for babel options when babelConfig is an object', functio
 	var fixturePath = fixture('es2015.js');
 	var fixtureSource = fs.readFileSync(fixturePath, 'utf8');
 
-	var options = babelConfigHelper.build(babelConfig, fixturePath, fixtureSource);
+	var options = babelConfigHelper.build(babelConfig, undefined, fixturePath, fixtureSource);
```

Comment:
I'd rather give this `undefined` a variable name (like `var disablePowerAssert = undefined`) so people looking at this later know what it's for.